### PR TITLE
Chore/nvm support

### PR DESCRIPTION
**Pull request by escapedcat**
Support for [nvm](https://github.com/nvm-sh/nvm#intro) helps people like me on their first install.  
It will automatically use node v12 if you are using nvm.  
Updated `engines` to `12` because otherwise you get i.e.:
```sh
error postcss-cli@9.1.0: The engine "node" is incompatible with this module. Expected version ">=12". Got "10.22.0"
error Found incompatible module.
```

Everything works like before if you don't use nvm. It's fine.  

Minor format changes on the README due to prettier-plugin. I can revert these if wanted. 

Update:  
Updated version in `.nvmrc` to `14` because it's already used here: https://github.com/getAlby/lightning-browser-extension/blob/master/.github/workflows/build.yml#L23

### CODE REVIEW

**Comment by bumi** [comment]
Thaks for the pull request. I am using nodenv myself. 
With the nvmrc file we force people to use node 12? is that a good practice? or can we somehow say something like `>=12` ?

**Comment by escapedcat** [comment]
> I am using nodenv myself.

Ah ok, haven't used it. How does it infer the noder version from this repo? 

> With the nvmrc file we force people to use node 12? is that a good practice? or can we somehow say something like `>=12` ?

Most projects using nvm I know are using it to indicate the version which is being used in production/testing/project runs stable. Not necessarly the lowest or highest supported version. I.e. https://github.com/facebook/react/blob/main/.nvmrc  

Using `12` indicates that in general this project should work with node anything node 12. When running `nvm install` latest v12 should be installed. Running only `nvm use` will use whatever v12 you have installed already.  
nvm users can always switch to a different version or not run `nvm use` to keep using their global version.  

Does this help?

**Comment by bumi** [comment]
@escapedcat sorry I forgot about this. 
thanks for the explanation. I think 14 is better than 12 (12 is pretty old). The issue I have with a fixed version is that it makes it hard for people to use other versions even though other versions are supported. 
I for example run node version 16.

nodenv uses a `.node-version` file. So it is actually pretty similar it seems. 

Guess the only question is if those files should be in git?

**Comment by escapedcat** [comment]
> The issue I have with a fixed version is that it makes it hard for people to use other versions even though other versions are supported. I for example run node version 16.

Got it. In my experience `.nvmrc` is indicating the minimum version. People using nvm are used to this. I'm not worried there anything hard to it for these people. They know they can run i.e. `nvm use 16` or have set a global version.

Looking at the [nvm-docs](https://github.com/nvm-sh/nvm#nvmrc) we could also use i.e.:
> to default to the latest LTS version
```
lts/*
```


> nodenv uses a `.node-version` file. So it is actually pretty similar it seems.
> Guess the only question is if those files should be in git?

Yeah, I think this would be helpful for other people using nodenv.

**Comment by bumi** [comment]
>  indicating the minimum version.

ok, that sound good to me. I just dislike forcing people to use a very specific version (e.g. if having 16.11.1 installed but the version file says 16.11.0)

**Review by bumi — APPROVED** [review]

**Comment by escapedcat** [comment]
Added node-version as well and update general node version to indicate >=16

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=10.0.0",
+    "node": ">= 16",
```
[the remark this answers is not in the document]
Comment (escapedcat) [reply package.json]: Ah, ok, good to know. Switched it to 14.

**Comment by escapedcat** [comment]
fyi this is how react is using it https://github.com/facebook/react/blob/main/.nvmrc

**Comment by bumi** [comment]
> fyi this is how react is using it https://github.com/facebook/react/blob/main/.nvmrc

so also 14. - but they force a specific patch level, which I think is not so nice if not needed

thank.

**Comment by escapedcat** [comment]
If people run into any issues with this we can adjust or remove

**Pull request merged**